app/util/mapLayerUtils.js

```diff
@@ -42,7 +42,10 @@ export const isFeatureLayerEnabled = (
   if (!Object.keys(mapLayers).includes(layerName)) {
     return false;
   }
-  const featureType = (feature.properties.type || '').toLocaleLowerCase();
+  const featureType =
+    layerName === 'roadworks'
+      ? null
+      : (feature.properties.type || '').toLocaleLowerCase();
```

Comment:
I implemented this temporarily to make the polyline click work. Can you please tell me where the map layer configuration is done and how I can add featureTypes?

Reply:
i will take a look.